git_cc/common.py

```diff
@@ -226,6 +226,7 @@ def path(path, args='-m'):
 cfg.read()
 CC_DIR = path(cfg.get(CFG_CC))
 DEBUG = cfg.getCore('debug', True)
+DEBUG = DEBUG == str(True)
```

Comment:
Hey @oplier , thanks for the PR. Two nits. 

1. You mind making it a single line
2. My python is terrible, but would it be safer to do `str(cfg.getCore('debug', True)) == str(True)`?